relabel/relabel.go

```diff
@@ -88,6 +88,22 @@ func relabel(labels model.LabelSet, cfg *config.RelabelConfig) model.LabelSet {
 			}
 		}
 		labels = out
+	case config.RelabelLabelDrop:
+		out := make(model.LabelSet, 0)
```

Comment:
Ok. I avoided that as I was a concerned about the potential waste of memory. Pushed update